services/claim-backend/src/modules/claim/claim.module.ts

```diff
@@ -30,15 +32,23 @@ import { BlockchainModule } from '@taraxa-claim/blockchain';
         maxRedirects: 5,
       }),
     }),
-    BlockchainModule,
+    GraphQLRequestModule.forRootAsync(GraphQLRequestModule, {
+      imports: [ConfigModule],
+      inject: [ConfigService],
+      useFactory: (config: ConfigService) => {
+        return {
+          endpoint: config.get<string>('reward.graphQLClaimUrl'),
+        };
+      },
+    }),
   ],
   controllers: [
     BatchController,
     RewardController,
     AccountController,
     ClaimController,
   ],
-  providers: [ClaimService],
-  exports: [ClaimService],
+  providers: [ClaimService, GraphQLService],
+  exports: [ClaimService, GraphQLService],
```

Comment:
No need to export it. We're using it in the same module

Reply:
Removed